src/Microsoft.Identity.Client/ApiEvent.cs

```diff
@@ -72,6 +72,7 @@ public string UserId
         public bool WasSuccessful
         {
             set => this["was_successful"] = value.ToString();
+            get => this["was_successful"] == "True";
```

Comment:
We were going for that, but then we saw you had List<Dictionary<String, String>> that solves the purpose too.